### PR TITLE
Adds number of notifications badge

### LINKED ISSUE #333: Show number of notifications
### Description
Show the number of notification on the right upper side of the notifications toggle button.

### PR DESCRIPTION
**Pull request by manuelwedler**
Closes #333

### CODE REVIEW

File: src/app/components/header/header.component.spec.ts [hunk 1]
```diff
@@ -154,4 +158,22 @@ describe('HeaderComponent', () => {
...
+        ).toBeTruthy();
+    });
+
+    it('should display the number of notifications in a badge', () => {
+        notificationService.addInfoNotification({
+            title: 'Testing',
+            description: 'Currently testing the application.',
+            icon: '',
+        }),
+            fixture.detectChanges();
```
Comment (nephix) [inline src/app/components/header/header.component.spec.ts:174]: Some weird formatting here, are tests not linted?

File: src/app/services/notification.service.spec.ts [hunk 2]
```diff
@@ -213,6 +213,30 @@ describe('NotificationService', () => {
         }
     ));
 
+    it('should emit the number of notifications', inject(
+        [NotificationService],
+        (service: NotificationService) => {
+            let expectatedValue = 0;
```
Comment (nephix) [inline src/app/services/notification.service.spec.ts:219]:
```suggestion
            let expectedValue = 0;
```

File: src/app/components/header/header.component.spec.ts (hunk 1 above)
Comment (manuelwedler) [reply src/app/components/header/header.component.spec.ts:174]: They are. Weird that this was not found

File: src/app/services/notification.service.spec.ts (hunk 2 above)
Comment (manuelwedler) [reply src/app/services/notification.service.spec.ts:219]: Thanks :)

**Pull request merged**